src/components/ec-table-filter/ec-table-filter.vue

```diff
@@ -58,11 +63,22 @@ const props = defineProps({
     type: String,
     default: 'Clear filters',
   },
+  isClearButtonHidden: {
+    type: Boolean,
+    default: false,
+  },
 });
 
 const emit = defineEmits(['update:modelValue', 'change']);
 
+const shownFilters = computed(() => (props.filters ? props.filters.filter(filter => !filter.isHidden) : []));
 const hasFilters = computed(() => !!Object.keys(props.modelValue).length);
+// eslint-disable-next-line no-underscore-dangle
+const haveOnlyTextFilter = computed(() => (shownFilters.value[0].component.__name === 'ec-text-filter') && props.isClearButtonHidden);
```

Comment:
let's move this to filter prop as well, so we could have any filter full width

Reply:
prop for filter could have name isFullWidth

Reply:
okay!